.github/workflows/pull-request-checker.yml

```diff
@@ -36,13 +36,14 @@ jobs:
       - name: Build Project # Build with npm
         run: npm run build
 
+      # Performs the project analysis using SonarQube Scanner.
       - uses: sonarsource/sonarqube-scan-action@v3
         env:
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }} # Authentication token required to access the SonarQube server
           SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
-      # If you wish to fail your job when the Quality Gate is red, uncomment the
-      # following lines. This would typically be used to fail a deployment.
```

Comment:
bu bilgilendirici bir yorumdu, ileride tersi yapılmak istenirse bu satırlar comment'e alınmalı diye değiştirilebilir silmektense

Reply:
o şekilde bıraktım ve söylediğin diğer kısımları düzelttim